### PR TITLE
Metadata and fav-icon

### PR DESCRIPTION
**Pull request by vibhiquartech**
https://apps.nrs.gov.bc.ca/int/jira/browse/CTHUB-298

### CODE REVIEW

**Comment by naomiaro** [comment]
Sorry @vibhiquartech I just merged some PRs, that must have caused another conflict here

**Review by naomiaro — CHANGES_REQUESTED** [review]
needs rebase

**Comment by naomiaro** [comment]
Since Sonarcloud is complaining about duplication here: https://github.com/nfl/react-helmet#reference-guide let's look into using nested templates and `titleTemplate`

**Comment by naomiaro** [comment]
This also needs another rebase to not `pick` commits by Kuan and Tim. They are already merged into the main branch.

**Comment by vibhiquartech** [comment]
Do you suggest i should fetch upstream and rebase to my fork as i have previously rebased to the bcgov/release-1.1.0?
________________________________

**Comment by naomiaro** [comment]
Just going to merge with duplication for now since I think part of the problem is that we're reusing page titles.

**Comment by naomiaro** [comment]
Hmm but now I see that only administrators will be allowed to merge if I do this.

File: frontend/src/pages/HouseholdForm.js [hunk 1]
```diff
@@ -21,6 +22,10 @@ const HouseholdFormPage = () => {
 
   return (
     <div>
+      <Helmet>
+        <title>Passenger vehicle rebate application form – CleanBC Go Electric</title>
+      </Helmet>
+      Hello BCeID {decoded.preferred_username}
```
Comment (naomiaro) [inline frontend/src/pages/HouseholdForm.js:28]: @vibhiquartech I think this decoded part might have been from the rebase, this variable doesn't exist anymore

**Review by naomiaro — CHANGES_REQUESTED** [review]

**Pull request closed without merging**